modules/understanding-idp.adoc

```diff
@@ -39,6 +39,11 @@ You can configure the following types of identity providers:
 |HTPasswd
 |Configure an `htpasswd` identity provider for a single, static administration user. You can log in to the cluster as the user to troubleshoot issues.
 
+[IMPORTANT]
+====
+The `htpasswd` identity provider option is included only to enable the creation of a single, static administration user. `htpasswd` is not supported as a general use identity provider for {product-title}. For the steps to configure the single user, see _Configuring an HTPasswd identity provider_.
```

Comment:
Same comments as for [config-idp.adoc](https://github.com/openshift/openshift-docs/pull/42822/files#diff-35db2c8995815f7c466cd4dab6f0c13698b5e30a7e8c9b04e09f843c1dfea76b)

Reply:
I have resolved those points throughout the PR.